README.md

```diff
@@ -81,6 +82,8 @@ To request support please create a [new discussion](https://github.com/kjaymille
 - [Flask Azure Container Apps Postgres Addon  Azure Container Apps](https://github.com/Azure-Samples/azure-flask-postgres-addon-aca)
 - [Flask CosmosDB - MongoDB Adapter  Azure Container Apps](https://github.com/Azure-Samples/azure-flask-cosmos-mongodb-aca)
 - [Flask CosmosDB - MongoDB Adapter  Azure App Service](https://github.com/Azure-Samples/azure-flask-cosmos-mongodb-appservice)
+- [Flask MySQL - Flexible Server Azure App Service](https://github.com/john0isaac/azure-flask-mysql-flexible-appservice)
```

Comment:
I think this is a temporary good choice to include working samples. In the long term. There should be samples that are created in the Azure-Samples Repo